pkg/provisioner/controller.go

```diff
@@ -389,7 +376,9 @@ func (c *Controller) handleDeleteClaim(key string) error {
 			return fmt.Errorf("error deleting objectBucket %v", ob.Name)
 		}
 	}
-	return nil
+
+	// delete all generated kubernetes resources
+	return c.deleteResources(ob, cm, secret)
```

Comment:
the delete on the OB is redundant with line 372.  The reason the 2nd delete call doesn't fail is that we ignore NotFound errors.  The result is an inaccurate log statement

Reply:
Good point! I'll delete L372-378 and let `deleteResources` do all the deletes.